woocommerce/woocommerce-ios
# Adding a package UI - Scaffolding for saved and carrier packages list

Part of #13551


## Description


Basic UI for saved packages and carrier list.

Features remaining for the "Saved" tab:
- Use proper saved data
- Add logic to save and delete saved package
- Decide how to show packages in the list (carrier and then saved or some other way)

Features remaining for the "Carrier" tab:
- Use proper saved data
- Add logic to save and delete saved package

Rest of the functionalities will be done in **separate** stacked PRs (done on top of this branch or trunk if this gets merged before).

## Testing information


1. Install and set up the Woo Shipping extension on your store.
2. Build and run the app with the `revampedShippingLabelCreation` feature flag enabled.
3. Create an order with the processing status and at least one physical product.
4. In the order details, select "Create Shipping Label."
5. Tap "Select a Package" button
6. Switch between Carrier and Saved tab
7. "Add package" button should be disabled until you select one of the packages from the Carrier or Saved list

## Screenshots


https://github.com/user-attachments/assets/61d933f1-7a5e-4edb-9752-6fb6e7b4494d

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
